add git stash indication

ev-agelos opened the pull request:
The icon doesn't look nice with the rest I just chose one :/
And I am not sure for the color too :D

Commit:
add git stash indication

jedahan commented on plugins/git/plugin.zsh:
```diff
@@ -172,13 +176,17 @@ geometry_prompt_git_render() {
     fi
   fi
 
+  if git rev-parse --quiet --verify refs/stash >/dev/null; then
```
I would check `PROMPT_GEOMETRY_GIT_SHOW_STASHES` here first

ev-agelos replied:
Right. Ill update it

jedahan commented on plugins/git/plugin.zsh:
```diff
@@ -29,6 +32,7 @@ PROMPT_GEOMETRY_GIT_CONFLICTS=${PROMPT_GEOMETRY_GIT_CONFLICTS:-false}
 PROMPT_GEOMETRY_GIT_TIME=${PROMPT_GEOMETRY_GIT_TIME:-true}
 PROMPT_GEOMETRY_GIT_TIME_LONG_FORMAT=${PROMPT_GEOMETRY_GIT_TIME_LONG_FORMAT:-false}
 PROMPT_GEOMETRY_GIT_TIME_SHOW_EMPTY=${PROMPT_GEOMETRY_GIT_TIME_SHOW_EMPTY:-true}
+PROMPT_GEOMETRY_GIT_STASHES=${PROMPT_GEOMETRY_GIT_STASHES:-false}
```
can we name this `PROMPT_GEOMETRY_GIT_SHOW_STASHES` and set the default to `true` ?

ev-agelos replied:
Sure!

jedahan commented on plugins/git/plugin.zsh:
```diff
@@ -172,13 +176,17 @@ geometry_prompt_git_render() {
   fi
 
-  render+="$(prompt_geometry_git_branch) ${conflicts}${GEOMETRY_GIT_SEPARATOR}${time} $(prompt_geometry_git_status)"
+  render+="$(prompt_geometry_git_branch) ${conflicts}${GEOMETRY_GIT_SEPARATOR}${time} $(prompt_geometry_git_status) ${stashes}"
```
I'd like to see what this looks like before the status, with the other 'optional' parts.
Also we should include the space / GEOMETRY_GIT_SEPARATOR as part of the `stashes=` assignment so if stashes is unassigned we don't have a trailing space.

jedahan reviewed (approved):
Great addition! Do you have the time to update the PR with my suggestions or should I?

jedahan commented:
Trying to find a replacement for the pin icon that keeps simple, solid geometry - these seem nice:

# `●` `✦` `╸` `⚈`

ev-agelos commented:
I will update the PR with your suggestions, thanks for the review!
It is my 2nd contribution and I would like to to it from start to finish :D

Commit:
Update PR with review comments

Check first if flag is true before setting the stashes variable
Stashes now defaults to true
Change color from "yellow" to 144(to be less bright that the status symbol)
Change symbol from 📌 to ● (tried the other ones but I had issues with
spaces, not alligned well)
Render it before the "status"

jedahan commented:
testing today

jedahan commented:
Thanks for your patience and the contribution!

ev-agelos commented:
Awesome! Thanks a lot too for you guidance!
